assembly/mapr/src/main/resources/scripts/splice/hbase-env.sh

```diff
@@ -93,7 +93,7 @@ SPLICE_HBASE_MASTER_OPTS="$SPLICE_HBASE_MASTER_OPTS -Dsplice.spark.driver.extraL
 SPLICE_HBASE_MASTER_OPTS="$SPLICE_HBASE_MASTER_OPTS -Dsplice.spark.driver.extraClassPath=/opt/mapr/hbase/hbase1.1.8-splice/conf:/opt/mapr/hbase/hbase1.1.8-splice/lib/*:/opt/splice/default/lib/*"
```

Comment:
Do we need to modify the driver classPath also?

Reply:
Yes, we do